friends/src/components/AddFriend/AddFriend.js

```diff
@@ -0,0 +1,78 @@
+import React, { useState } from "react";
+import axios from "axios";
+
+const AddFriend = () => {
+  const [name, setName] = useState("");
+  const [age, setAge] = useState(null);
+  const [email, setEmail] = useState("");
+
+  const postFriend = () => {
+    const postObj = {
+      name: name,
+      age: age,
+      email: email
+    };
+
+    return axios
+      .create({
+        headers: {
+          authorization: sessionStorage.getItem("token")
+        },
+        username: "Lambda School",
+        password: "i<3Lambd4"
+      })
+      .post(`http://localhost:5000/api/friends`, postObj)
+      .then(res => {
+        console.log(res);
```

Comment:
Unsure of what this server returns, but typically a part of the response to signal that the post was successful can be returned and used on the front end